src/components/CategoryPicker/categoryPickerPropTypes.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
+import categoryPropTypes from '../categoryPropTypes';
+
+const propTypes = {
+    /** The report ID of the IOU */
+    reportID: PropTypes.string.isRequired,
+
+    /** The policyID we are getting categories for */
+    policyID: PropTypes.string,
+
+    /** The type of IOU report, i.e. bill, request, send */
+    iouType: PropTypes.string.isRequired,
+
+    /** Collection of categories attached to a policy */
+    policyCategories: PropTypes.objectOf(categoryPropTypes),
```

Comment:
Always helps if we can call out what are Onyx props
```suggestion

    /* Onyx Props */
    /** Collection of categories attached to a policy */
    policyCategories: PropTypes.objectOf(categoryPropTypes),
```

Reply:
Done - bcf2d43